script.js

```diff
@@ -171,5 +171,17 @@
             }
         }
     ])
-
+    
 })(angular, window, document)
+
+// When the user clicks on the button, scroll to the top of the document
+function topFunction() {
+  document.body.scrollIntoView(true) // For Safari
+  document.documentElement.scrollIntoView(true) // For Chrome, Firefox, IE and Opera
```

Comment:
Hmm, `document.body.scrollIntoView()` works for me in Chrome and Firefox. Any reason why we use `document.documentElement` instead?

Reply:
Not really.